detox/android/detox/src/full/java/com/wix/detox/adapters/server/WebSocketClient.java

```diff
@@ -54,13 +54,13 @@ public void connectToServer(String url, String sessionId) {
                 .build();
 
         final Request request = new Request.Builder().url(url).build();
-        this.websocket = client.newWebSocket(request, wsEventsHandler);
+        this.websocket = client.newWebSocket(request, wsEventListener);
 
         client.dispatcher().executorService().shutdown();
     }
 
     public void sendAction(String type, Map params, Long messageId) {
-        Log.i(LOG_TAG, "At sendAction");
+        Log.i(LOG_TAG, "Sending out action '" + type + "'");
 
```

Comment:
Do you mind if we print message if as well?

Reply:
Done